test/e2e/page-objects/pages/header-navbar.ts

```diff
@@ -71,13 +71,6 @@ class HeaderNavbar {
     await this.driver.clickElement(this.switchNetworkDropDown);
   }
 
-  async check_currentSelectedNetwork(networkName: string): Promise<void> {
-    console.log(`Validate the Switch network to ${networkName}`);
-    await this.driver.waitForSelector(
-      `button[data-testid="network-display"][aria-label="Network Menu ${networkName}"]`,
-    );
-  }
-
```

Comment:
I've re-ordered this alphabetically and took the opportunity to add documentation in a couple of functions that didn't have it